docs/_user_guide/data_transfer.md

```diff
@@ -1,339 +1,257 @@
-
-To copy files from Yen servers to your local machine, open a new terminal without connecting to the Yens. Use the `cd` command to navigate to the local directory where you want to copy the files to. Then run: 
+To upload a single file
+```title="LFTP Terminal Command"
+put filename.bla
+```
 
-```bash
-$ cd my_local_folder
-$ scp -r <SUNetID>@yen.stanford.edu:/zfs/projects/students/<my_project_dir>/results .
+To upload a batch of files
+```title="LFTP Terminal Command"
+mput *.csv
 ```
 
-In this example, we're copying the `results` folder from the Yen's ZFS file system to your local directory (`.` signifies the current directory). If you're copying files (not directories), omit the `-r` flag. To transfer multiple files, use the wildcard `*` to match several files.
+## Rclone
+Rclone is a command-line program used to manage files on cloud storage. It supports various cloud storage providers and allows users to transfer, sync, and backup data efficiently. Rclone is particularly useful for archiving data to cloud storage providers like Google Drive and Amazon S3.
 
-### Using Other Transfer Tools
-See <a href="/faqs/rclone.html" target="_blank">rclone</a> and <a href="/faqs/rclone.html" target="_blank">rsync</a> pages to learn two additional transfer tools. 
+Check out the our [**rclone blog**](/blog/2023/09/18/rclone-files-from-yens-to-google-drive/) post for more information on how to use rclone to transfer data to and from the yens.
 
----
-<a href="/gettingStarted/6_filesystem.html"><span class="glyphicon glyphicon-menu-left fa-lg" style="float: left;"/></a> <a href="/gettingStarted/8_jupyterhub.html"><span class="glyphicon glyphicon-menu-right fa-lg" style="float: right;"/></a>
+## General Guidelines
+| **File Size/Transfer Scenario**            | **Recommended Methods**                                                                 | **Explanation**                                                                                                                                                                |
+|--------------------------------------------|----------------------------------------------------------------------------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| **Files less than 1 GB**                   | Upload via JupyterHub or use `scp`.                                                    | For small files, simple methods like uploading directly through JupyterHub or using `scp` (Secure Copy Protocol) are quick, straightforward, and efficient.                   |
+| **Files between 1 GB and 10 GB**           | Use `scp` over the Data Transfer Node (`yen-transfer`).                                 | Transferring directly to the Data Transfer Node ensures faster speeds and reduces load on the interactive nodes. `scp` remains effective for these moderately sized files.     |
```

Comment:
moderately sized files, providing you have a fast and stable internet connection.